src/Maestro/Client/src/Microsoft.DotNet.Maestro.Client.csproj

```diff
@@ -12,7 +12,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Rest.ClientRuntime" Version="$(MicrosoftRestClientRuntimeVersion)" />
+    <PackageReference Include="Azure.Core" Version="1.0.1" />
```

Comment:
@alexperovich we made this change in my local, is it okay to tie it down to this particular version?